tools/project_manager/downloadproofed.php

```diff
@@ -45,6 +43,3 @@
 // SENDING PAGE-TEXT TO USER
 // It's a text/plain document, so no encoding is necessary.
 echo $data;
-
-// vim: sw=4 ts=4 expandtab
```

Comment:
so we are open to removing these! I'll take note for future merge requests.

Reply:
Yup, time for them to die.

Reply:
I've updated the Best Practices guide to reflect this. That page will need updating after we finally do the code standardization later too.